Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2222,51 +2225,41 @@ - (void)clearUnreadMessagesIndicator
     }
 }
 
-- (void)createScrollButtons
+- (void)createConversationScrollButtons
 {
-    self.scrollDownButton = [self createScrollButton:@"\uf103" selector:@selector(scrollDownButtonTapped)];
+    self.scrollDownButton = [[ConversationScrollButton alloc] initWithIconText:@"\uf103"];
+    [self.scrollDownButton addTarget:self
+                              action:@selector(scrollDownButtonTapped)
+                    forControlEvents:UIControlEventTouchUpInside];
+    [self.view addSubview:self.scrollDownButton];
+    [self.scrollDownButton autoSetDimension:ALDimensionWidth toSize:self.scrollDownButton.width];
```

Comment:
We're constraining it's width to it's current width? Maybe this is idiomatic, but I've never seen it before and it seems a little weird.

Should it be something like self.scrollDownButton.circleSize.width?

Reply:
Done in https://github.com/WhisperSystems/Signal-iOS/commit/2ec1e7e92560b2c5f222ab77d479c3839ed79ed4